Autopilot/Test/Src/DataStructures/Test_Vector_Class.cpp

```diff
@@ -0,0 +1,109 @@
+#include <gtest/gtest.h>
+#include "math.h"
+#include "vectorClass.hpp"
+
+
+using namespace std; 
+using ::testing::Test;
+
+TEST(VectorClassTest, DefaultConstructor){
+    /***SETUP***/
+    Vector3D vector;
+    /***ASSERTS***/
+    EXPECT_EQ(vector.getX(), 0);
+    EXPECT_EQ(vector.getY(), 0);
+    EXPECT_EQ(vector.getZ(), 0);
+}
+
+TEST(VectorClassTest, DefinedConstructor){
+    /***SETUP***/
+    Vector3D vector(1,1,1);
```

Comment:
same here. Make each of the coordinates diff values so we know the coordinates were initialized with the correct parameter